### PR TITLE
Use css variable for font size.

### PR DESCRIPTION
**Pull request by Carreau**
Those are minor change of size (12px to 13px), and 0.8em to 0.86,
thigh they will fix issues, like nbconvert slides trying to bump up the
size by updating the variables values.

### CODE REVIEW

**Comment by fcollonval** [comment]
please update documentation snapshots

**Comment by fcollonval** [comment]
Edit bot commit to remove bad snapshot update

**Review by fcollonval — APPROVED** [review]
Thanks @Carreau

**Comment by krassowski** [comment]
1. This closes https://github.com/jupyterlab/jupyterlab/issues/7394 and is relevant to the discussion in there
2. What do you think about preserving the old font size by using `calc`, e.g. `calc(12/13 * var(--jp-ui-font-size1))`? This way we get the benefit of customization, but keep the status quo of the font size in table being slightly smaller.

**Comment by krassowski** [comment]
This was my concern at first too, but I tested on Chrome and it works well. Also had a quick glance at known issues for cros--browser support at https://caniuse.com/calc and I do not see a reason for that to be a problem, but we probably should also test on Safari (because Safari).

**Comment by jtpio** [comment]
Looks like this picked up some conflicts.

File: packages/rendermime/style/base.css [hunk 1]
```diff
@@ -413,7 +413,7 @@
   border-spacing: 0;
   border: none;
   color: var(--jp-ui-font-color1);
-  font-size: 12px;
+  font-size: var(--jp-ui-font-size1);
```
Comment (krassowski) [inline packages/rendermime/style/base.css:416]:
```suggestion
  font-size: calc(12/13 * var(--jp-ui-font-size1));
```

**Review by krassowski — COMMENTED** [review]
This should allow to eliminate the visual changes to table size (for the most part).

File: packages/rendermime/style/base.css (hunk 1 above)
Comment (Carreau) [reply packages/rendermime/style/base.css:416]: the main question is do we _want_ to have table be 12/13 of the normal font size just for the sake of keeping a value that was chosen 6 years ago ? 

I haven't been involved in JupyterLab design in a long time, so I don't want my opinion to weight too much on this, but I believe that uniformity is better, and would personally avoid calc even if it make a slight change of theme

**Comment by fcollonval** [comment]
As we are planning a 3.4 release, I suggest to acknowledge the change from 12px to `jp-font-size1` (aka 13px per default) in that release.

**Comment by Carreau** [comment]
> As we are planning a 3.4 release, I suggest to acknowledge the change from 12px to `jp-font-size1` (aka 13px per default) in that release.

What would be the right place to do so ?

**Comment by fcollonval** [comment]
Merging in master...

>  What would be the right place to do so ?

I added this PR in the roadmap for 3.4.0 #12268

**Comment by Carreau** [comment]
Thanks !

**Comment by fcollonval** [comment]
@meeseeksdev please backport to 3.4.x

**Pull request merged**